plugins/jobtype/src/azkaban/jobtype/HadoopJobUtils.java

```diff
@@ -85,6 +85,12 @@
   // Azkaban built in property name
   public static final String JVM_ARGS = "jvm.args";
 
+  // MapReduce config for specifying additional namenodes for delegation tokens
+  public static final String MAPREDUCE_JOB_OTHER_NAMENODES = "mapreduce.job.hdfs-servers";
+
+  // Azkaban property for listing additional namenodes for delegation tokens
+  private static final String OTHER_NAMENODES_PROPERTY = "other_namenodes";
```

Comment:
Right now it's private in Azkaban core. If you want we can release a new version of Azkaban core which exports this property.